RichClipboard-Tests.package/RCClippingsListTest.class/instance/setUp.st

```diff
@@ -0,0 +1,6 @@
+running
+setUp
+	systemClipboard := RCSystemClipboardNullObject new.
+	storage := RCStorage new.
+	clippingsList := RCClippingsList newWithStorage: storage
+																	andSystemClipboard: systemClipboard
```

Comment:
Ich weiß gar nicht ob ich das so gut finde die instanzen widerzuverwenden. Dadurch beeinflussen sich die Tests potenziell gegenseitig, sprich ein gefailter Test kann dafür sorgen, dass ein anderer failed obwohl dessen Implementierung eigentlich funktioniert oder andersrum (z.B. wenn ein Test Clippings in den Storage hinzufügt aber der Cleanup failed o.ä.).

Reply:
Oder wird `setUp` und `tearDown` pro Test jedes mal neu ausgeführt? Wenn dem so ist, dann nevermind... :sweat_smile:

Reply:
wird pro Test neu ausgeführt